### ISSUE TITLE
Feature: Add me to the Speachlist Button at Motions

### ISSUE THREAD

**Issue by TheJoKlLa** [OpenSlides/OpenSlides#5200]
Offers a parallel reading and writing of personal notes, with the option to participate to the discussion.

**Comment by emanuelschuetze** [comment]
Good idea. We can do this for 4.x - currently there is a workaround to open (current) list of speakers in a new tab.
